app/views/documents/show/_actions.html.erb

```diff
@@ -53,15 +53,23 @@
                  text: "Undo withdrawal",
                  data_attributes: { gtm: "undo-withdraw" },
                  href: unwithdraw_path(@document) %>
+    <% elsif @edition.scheduled? %>
+      <%= form_tag create_preview_path(@edition.document) do %>
+        <%= render "govuk_publishing_components/components/button", text: "Preview" %>
+      <% end %>
+      <%= delete_draft_link %>
     <% else %>
       <%= form_tag submit_document_for_2i_path(@edition.document) do %>
         <%= render "govuk_publishing_components/components/button", text: "Submit for 2i review" %>
       <% end %>
 
       <%= render "govuk_publishing_components/components/button", text: "Preview", href: preview_document_path(@edition.document), secondary: true %>
 
-      <%= link_to "Publish", publish_confirmation_path(@edition.document), class: "govuk-link govuk-link--no-visited-state" %>
-
+      <% if @edition.schedulable? && current_user.has_permission?(User::PRE_RELEASE_FEATURES_PERMISSION) %>
```

Comment:
Do we need to be considering here whether it's in that 15 minute window?

Reply:
`#schedulable?` checks for that.